tests/test_url_for.py

```
@@ -0,0 +1,56 @@
+    (
+        ("test_app.page", "/", {}),
+        (test_app_page.__name__, "/", {}),
+        ("test_app.calculator.page", "/calculator/", {}),
+        (calculator_page.__name__, "/calculator/", {}),
+        ("test_app.user.page", "/user/", {}),
+        (user_page.__name__, "/user/", {}),
+        ("test_app.user._id_.page", "/user/1/", {"id": 1}),
+        (user_by_id_page.__name__, "/user/1/", {"id": 1}),
+        ("test_app.user._id_.page", "/user/C0FF33/", {"id": "C0FF33"}),
+        (user_by_id_page.__name__, "/user/C0FF33/", {"id": "C0FF33"}),
+    ),
```

Ruff (0.14.1)

Wrong values type in `pytest.mark.parametrize` expected `list` of `tuple`

Use `list` of `tuple` for parameter values

(PT007)